python/ray/autoscaler/resource_demand_scheduler.py

```diff
@@ -87,9 +87,13 @@ def calculate_node_resources(
 
         def add_node(node_type):
             if node_type not in self.node_types:
-                raise RuntimeError("Missing entry for node_type {} in "
-                                   "available_node_types config: {}".format(
-                                       node_type, self.node_types))
+                logger.warn(
+                    f"Missing entry for node_type {node_type} in "
+                    f"cluster config: {self.node_types}. This "
+                    f"node's resources will be ignored. If you are using an "
+                    f"unmanaged node, set the user_node_type tag manually in "
+                    f"your cloud provider.")
```

Comment:
what do you mean by "your cloud provider" here? is this for:
1) custom cloud provider 
2) cluster_config["provider"]?

Reply:
clarified